Modelica/Media/R134a.mo

```diff
@@ -1943,12 +1943,10 @@ The function cannot be inverted in a numerical way. Please use functions <a href
         if liquid then
           damping := 0.3;
           //damping on the liquid side
-          d := R134aData.data.FDCRIT*Common.CubicSplineEval(localx, dl_coef[int,
-            1:4])*1.02;
+          d := R134aData.data.FDCRIT*Common.CubicSplineEval(localx, dl_coef[int,1:4])*1.5;
```

Comment:
So the main change really is using 1.5 instead of 1.02?

As long term solution, the Newton iteration should probably be bounded and have asserts in it for the slope.
Also, we should have two tests: a Modelica test comparing forward/backward consistency of the Newton iterations; and a python script comparing values against RefProp (from python, the most convenient way is to use CoolProp to call RefProp).

Reply:
> So the main change really is using 1.5 instead of 1.02?

Yes, it is.